minard/tools.py

```diff
@@ -10,3 +17,86 @@ def parseiso(timestr):
     dt = datetime.strptime(timestr,'%Y-%m-%dT%H:%M:%S.%fZ')
     return calendar.timegm(dt.timetuple()) + dt.microsecond/1e6
 
+def import_HLDQ_runnumbers():
+    server = couchdb.Server("http://snoplus:"+os.environ["COUCHDB_PASSWORD"]+"@couch.snopl.us")
+    dqDB = server["data-quality"]
+    runNumbers = []
+    for row in tellieDB.view('_design/data-quality/_view/runs'):
+        runNum = int(row)
+        if runNum not in runNumbers:
+            runNumbers.append(runNum)
+    return runNumbers
+
+def import_TELLIE_runnumbers():
+    server = couchdb.Server("http://snoplus:"+os.environ["COUCHDB_PASSWORD"]+"@couch.snopl.us")
+    tellieDB = server["telliedb"]
+    runNumbers = []
+    for row in tellieDB.view('_design/ratdb/_view/select_time'):
+        if row.key[0] != "TELLIE_RUN":
+            continue
+        runNum = int(row.key[1])
+        if runNum not in runNumbers:
+            runNumbers.append(runNum)
+    return runNumbers
+
+def import_TELLIEDQ_ratdb(runNumber):
+    server = couchdb.Server("http://snoplus:"+os.environ["COUCHDB_PASSWORD"]+"@couch.snopl.us")
+    dqDB = server["data-quality"]
+    data = None
+    for row in dqDB.view('_design/data-quality/_view/runs'):
+        if(int(row.key) == runNumber):
+            runDocId = row['id']
+            try:
+                print(runDocId)
+                data = dqDB.get(runDocId)["checks"]["dqtellieproc"]
+            except KeyError:
+                print("Got key error: %d" % runNumber)
```

Comment:
These won't be logged by the web server anywhere. If you want to log them I think you can use `app.logger.warning()`, although if they are just used for testing I think it's ok to leave it as is.